airbyte-ci/connectors/metadata_service/lib/README.md

```diff
@@ -34,3 +34,17 @@ poetry run pytest
 ```bash
 poetry run metadata_service validate tests/fixtures/valid/metadata_registry_override.yaml
 ```
+
+## Useful Commands
+
+### Replicate Production Data in your Development Bucket
+This will replicate all the production data to your development bucket. This is useful for testing the metadata service with real up to date data.
+```bash
+TARGET_BUCKET=<YOUR-DEV_BUCKET> poetry poe replicate-prod
+```
+
+### Promote Connector Version to Latest
+This will promote the specified connector version to the latest version in the registry. This is useful for testing a connector prerelease version in your development bucket before promoting it to the latest version in the registry.
```

Comment:
```suggestion
This will promote the specified connector version to the latest version in the registry. This is useful for creating a mocked registry in which a prerelease connector is treated as if it was already published. 
```